src/OpenStack.Net/OpenStack/Services/ContentDelivery/V1/IContentDeliveryService.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace OpenStack.Services.ContentDelivery.V1
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// This is the base interface for the OpenStack Content Delivery Service V1.
+    /// </summary>
+    /// <seealso href="">OpenStack Content Delivery API V1 Reference</seealso>
+    /// <preliminary/>
+    public interface IContentDeliveryService : IHttpService, IExtensibleService<IContentDeliveryService>
+    {
+        #region Service Information
+
+        Task<GetHomeApiCall> PrepareGetHomeAsync(CancellationToken cancellationToken);
```

Comment:
:exclamation: This needs to be documented (along with several other cases in the same file).